src/components/Typography/Typography.stories.mdx

```diff
@@ -128,58 +170,75 @@ The following CSS variables are available through the DS.
 ```
 
 > **RECOMMENDATIONS**  
-> When making font selections for project based components outside the DS, sans serif fonts display better on computers and mobile devices. Avoid decorative or overly stylized fonts, as they are often difficult to read even for users without visual impairments or reading Standard Text Components
-
-<Story name="Font Colors" />
+> When making font selections for project based components outside the DS, sans serif fonts display better on computers and mobile devices. Avoid decorative or overly stylized fonts, as they are often difficult to read even for users without visual impairments or reading Standard Text Default
 
 ## Font Colors
 
 All text within a project, unless specifically noted in mockups, should be black on a white background.
 
-In cases where specific colors must be applied to text elements, the DS provides CSS variables for the full NYPL color palette. Refer to the [Colors](https://nypl.github.io/nypl-design-system/storybook-static/?path=/docs/style-guide-colors--colors-brand) section of the Style Guide for details.
+<Preview>
+  <Story name="Font Colors">
+    <>
```

Comment:
I see what you meant about the loose text not working inside a `Preview`. I was going to suggest this but glad you found it.